### PR TITLE
finallizing comments/doc-comments

### PR DESCRIPTION
**Pull request by ramico**
-no code changes

### CODE REVIEW

File: includes/class-wc-siftscience-html.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * This class henarates html to display subsections stated in the admin file.
+ * This class henarates displayes html for subsections as stated in the admin file.
```
Comment (nabsul) [inline includes/class-wc-siftscience-html.php:3]: This class generates html for subsections as...

File: includes/class-wc-siftscience-element.php [hunk 2]
```diff
@@ -54,10 +52,10 @@ public function __construct( WC_SiftScience_Logger $logger ) {
 		 * desc_tip Mixed [string]
 		 *     field type of checkbox; the desc_tip text is going underneath control
 		 *     field type of select, number or text; a question mark pop-up appears before control with desc_tip text
-		 * desc_tip is added in element options [X] sanitized.
+		 * desc_tip is added in element options [X] sanitized to fasle.
 		 *
 		 * @param string $type            Element type name.
-		 * @param string $id              HtmlElement ID.
+		 * @param string $id              HTMLElement ID.
```
Comment (nabsul) [inline includes/class-wc-siftscience-element.php:58]: HTML Element (add space between the two words.

File: includes/class-wc-siftscience-element.php [hunk 3]
```diff
@@ -54,10 +52,10 @@ public function __construct( WC_SiftScience_Logger $logger ) {
 		 * desc_tip Mixed [string]
 		 *     field type of checkbox; the desc_tip text is going underneath control
 		 *     field type of select, number or text; a question mark pop-up appears before control with desc_tip text
-		 * desc_tip is added in element options [X] sanitized.
+		 * desc_tip is added in element options [X] sanitized to fasle.
```
Comment (nabsul) [inline includes/class-wc-siftscience-element.php:55]: Please fix indentation. also "false"  at the end has spelling mistake.

File: includes/class-wc-siftscience-element.php [hunk 4]
```diff
@@ -14,9 +14,7 @@
 if ( ! class_exists( 'WC_SiftScience_Element' ) ) :
 
 	/**
-	 * Helper class for generating html
-	 *
-	 * Class WC_SiftScience_Element
+	 * Class for adding WoocCmmearce elements.
```
Comment (nabsul) [inline includes/class-wc-siftscience-element.php:17]: WooCommerce

File: includes/class-wc-siftscience-element.php [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * This class creates ec element according to woocommearce library soecs.
+ * Class for creating wc elements according to WoocCmmearce library.
```
Comment (nabsul) [inline includes/class-wc-siftscience-element.php:3]: WooCommerce

**Review by nabsul — CHANGES_REQUESTED** [review]

File: includes/class-wc-siftscience-html.php (hunk 1 above)
Comment (ramico) [reply includes/class-wc-siftscience-html.php:3]: agreed

File: includes/class-wc-siftscience-element.php (hunk 3 above)
Comment (ramico) [reply includes/class-wc-siftscience-element.php:55]: this line should not be indented could however have a new line above  
spilling will be corrected

File: includes/class-wc-siftscience-element.php (hunk 2 above)
Comment (ramico) [reply includes/class-wc-siftscience-element.php:58]: not arguing but i think this saying it's a DOM thing

**Review by nabsul — APPROVED** [review]

**Pull request merged**